docs/src/examples/quick-start.md

```diff
@@ -0,0 +1,73 @@
+# Quick Start
+
+## Prerequisites
+
+Version Requirements:
+
+- JDK: >= 1.8
+- Maven: >= 3.8.4 (TODO: lower the required version)
+- Java Client: >= v3.1.0
+- TiKV: >= v4.0.0
```

Comment:
I think TiKV's version should be larger than 5.0.0 because RawKV API like CAS and TTL is not implemented in TiKV v4

Reply:
I wrote the required TiKV version to 4.0 for the following reasons:
1. there are some users using TiKV 4.0 and lower versions
2. there isn't any examples of TTL and CAS in this document